### DISCUSSION TITLE
Apple M1 Processor?

### DISCUSSION THREAD

**Discussion by bradmesserle** [beehive-lab/TornadoVM#51] [Q&A] [answered]
Has anyone tried running this with the new M1 Processor?

**Comment by jjfumero** [comment]
No, we haven't tried yet. 
All accelerators that we have tried are summarized here:

Drivers supported:
https://github.com/beehive-lab/TornadoVM/blob/master/assembly/src/docs/TORNADOVM_TESTED_DRIVERS.md 

Devices:
https://github.com/beehive-lab/TornadoVM/blob/master/assembly/src/docs/1_INSTALL.md#installing-tornadovm

**Comment by stratika** [comment, marked as answer]
> Has anyone tried running this with the new M1 Processor?

Hi @bradmesserle, I tried it last week and we uploaded a [blog](https://www.tornadovm.org/post/install-tornadovm-on-apple-m1-pro) that highlights some initial findings and guidelines for installation of TornadoVM on Apple MacBook Pro with the M1 Pro chip. Let us know if you have any particular issue, that we can have a look. As mentioned in the blog, there are several features that seem to be not supported.
